src/components/views/elements/AppTile.js

```diff
@@ -329,9 +329,7 @@ export default class AppTile extends React.Component {
      * Called when widget iframe has finished loading
      */
     _onLoaded() {
-        if (!ActiveWidgetStore.getWidgetMessaging(this.props.id)) {
-            this._setupWidgetMessaging();
-        }
+        this._setupWidgetMessaging();
```

Comment:
Note to reviewers: this was causing Jitsi to stop receiving messages after going to a different room and back. I'm not sure if removing this check has any other adverse effects but so far I've noticed none.